openmdao/utils/coloring.py

```diff
@@ -702,14 +702,20 @@ def display(self, show=True, fname=_default_coloring_imagefile):
             Path to the location where the plot file should be saved.
         """
         try:
-            from matplotlib import pyplot, cm
+            import matplotlib as mpl
+            from matplotlib import pyplot
+            mplLessThan360 = True if float(mpl.__version__.replace('.', '')) < 360 else False
```

Comment:
You might want to take a look at Version in packaging.version, as it can give a more readable version comparison than replacing the period and turning the version into a float (e.g., is our target 3.6.0, or 36.0?). We use this a number of places in the tests, and a few places in the code too. The check looks like: `if Version(np.__version__) >= Version("1.14"):`